person.rb

```diff
@@ -0,0 +1,32 @@
+class Person
+  attr_reader :id, :parent_permission
+  attr_accessor :age, :name
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1...10_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    return true if age >= 18
+
+    false
+  end
```

Comment:
[OPTIONAL] To keep your code simple, you may use:
```ruby
def of_age?
    @age >= 18
  end
```
of_age will return true if @age is greater or equal to 18 and false otherwise.